addons/mail/static/src/core_ui/message_reaction_menu.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionMenu" owl="1">
+        <Dialog size="'sm'" header="false" footer="false"
+            contentClass="'o-mail-MessageReactionMenu h-50 d-flex'">
+            <div class="d-flex h-100" t-on-keydown="onKeydown" t-ref="root">
+                <div class="d-flex overflow-auto flex-column bg-200 p-2 h-100">
+                    <t t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content">
+                        <button class="btn p-1 rounded-2 mx-3" t-att-class="{ 'bg-300 border-primary': reaction === state.reaction }"
+                            t-att-title="getEmojiShortcode(reaction)" t-att-data-reaction="reaction" t-on-click="() => state.reaction = reaction">
+                            <span class="mx-1" t-esc="reaction.content"/>
+                            <span class="mx-1" t-att-class="{ 'text-primary': reaction === state.reaction }" t-esc="reaction.count"/>
+                        </button>
+                    </t>
+                </div>
+                <div class="d-flex overflow-auto flex-column flex-grow-1 bg-100 p-2 h-100">
+                    <div t-foreach="state.reaction.personas" t-as="persona" t-key="persona.id" class="d-flex p-1 align-items-center">
+                        <img class="rounded-circle o_object_fit_cover o-mail-MessageReactionMenu-avatar" t-att-src="threadService.avatarUrl(persona, props.message.originThread)"/>
+                        <span class="d-flex flex-grow-1 align-items-center">
+                            <span class="mx-2 text-truncate fs-6" t-esc="persona.name"/>
+                            <div class="flex-grow-1"/>
+                            <i t-if="this.store.self === persona" class="fa fa-trash" title="Remove" t-on-click.stop="() => this.messageService.removeReaction(state.reaction)"/>
```

Comment:
Better be a `<button>`, so still is better. Also should have `opacity: 0` in desktop when not mouse hovering.